.bazelrc

```diff
@@ -19,6 +19,7 @@ build --ios_simulator_version=15.2
 build --verbose_failures
 build --workspace_status_command=envoy/bazel/get_workspace_status
 build --xcode_version=13.2.1
+build --define=apple.experimental.tree_artifact_outputs=1
```

Comment:
This didn't seem to affect the distribution (see [CI run](https://github.com/envoyproxy/envoy-mobile/runs/6406314786?check_suite_focus=true)), but it did break the `framework_imports_extractor` rule.

I could use your help to update that if you have the time.

Reply:
can we make rules_xcodeproj pass this as needed instead?

Reply:
i think we can otherwise read this in the rule, and branch on that to just re-propagate the same files in that rule tho, but in general i think the zip is normally preferred just for ease of moving it around

Reply:
I was able to remove this and get around that `archived_bundles_allowed` issue by removing the Objective-C app from the Xcode project. I'll follow up on fixing that in rules_xcodeproj.